### PR TITLE
Support setting attribute values for dropdown & multiselect.

### PR DESCRIPTION
**Pull request by piotrgrundas**
[Saleor PR](https://github.com/mirumee/saleor/pull/7037)

image
image

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [x] All visible strings are translated with proper context.
1. [x] All data-formatting is locale-aware (dates, numbers, and so on).
1. [x] Translated strings are extracted.
1. [x] Number of API calls is optimized.
1. [x] The changes are tested.
1. [x] Data-test are added for new elements.
1. [x] Type definitions are up to date.
1. [x] Changes are mentioned in the changelog.
1. [x] The changes are tested in different browsers (Chrome, Firefox, Safari).
1. [x] The changes are tested in light and dark mode.

### Test environment config

API_URI=https://master.staging.saleor.cloud/graphql/

### CODE REVIEW

**Comment by dominik-zeglen** [comment]
Not ringing any bells what it should be doing. Can you point to clickup ticket and link affected storybook pages?

**Comment by piotrgrundas** [comment]
@dominik-zeglen It's required for external integration.

**Review by orzechdev — COMMENTED** [review]
@piotrgrundas I don't understand what do you mean by "it's required for external integration". It doesn't explain to me any changes added here.

Also, name of the commit `#` doesn't explain anything.

File: src/attributes/components/AttributeValues/AttributeValues.tsx [hunk 1]
```diff
@@ -113,10 +119,13 @@ const AttributeValues: React.FC<AttributeValuesProps> = ({
...
                 <TableCell className={classes.columnAdmin}>
-                  {maybe(() => value.slug) ? value.slug : <Skeleton />}
+                  {value?.slug ?? <Skeleton />}
+                </TableCell>
+                <TableCell className={classes.columnStore}>
+                  {value?.value ?? <Skeleton />}
                 </TableCell>
                 <TableCell className={classes.columnStore}>
-                  {maybe(() => value.name) ? value.name : <Skeleton />}
+                  {value?.name ?? <Skeleton />}
```
Comment (dominik-zeglen) [inline src/attributes/components/AttributeValues/AttributeValues.tsx:128]: I thought I'd never see this operator 😮

**Review by dominik-zeglen — APPROVED** [review]

File: src/attributes/components/AttributeValueEditDialog/AttributeValueEditDialog.tsx [hunk 2]
```diff
@@ -43,10 +45,11 @@ const AttributeValueEditDialog: React.FC<AttributeValueEditDialogProps> = ({
 }) => {
   const intl = useIntl();
   const initialForm: AttributeValueEditDialogFormData = {
-    name: maybe(() => attributeValue.name, "")
+    name: maybe(() => attributeValue.name, ""),
```
Comment (jwm0) [inline src/attributes/components/AttributeValueEditDialog/AttributeValueEditDialog.tsx:48]: use this opportunity to get rid off `maybe`

**Review by jwm0 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**